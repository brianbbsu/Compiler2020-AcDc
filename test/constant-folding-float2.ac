f a
a = 123456.78901 + 987654.43212
p a
