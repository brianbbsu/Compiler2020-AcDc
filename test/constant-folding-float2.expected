1111111.22113
